metadata.yaml

```diff
@@ -28,7 +29,7 @@ assumes:
 provides:
   admin:
     interface: temporal
-    limit: 1
+    limit: 4
```

Comment:
What does the admin relation provide in this case?

Reply:
Good question, I've added the reasoning behind this to the PR description.